plugins/devtools/src/api/DevToolsClient.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { DiscoveryApi, IdentityApi } from '@backstage/core-plugin-api';
+import {
+  ConfigInfo,
+  DevToolsInfo,
+  ExternalDependency,
+} from '@backstage/plugin-devtools-common';
+import { ResponseError } from '@backstage/errors';
+import { DevToolsApi } from './DevToolsApi';
+
+export class DevToolsClient implements DevToolsApi {
+  private readonly discoveryApi: DiscoveryApi;
+  private readonly identityApi: IdentityApi;
+
+  public constructor(options: {
+    discoveryApi: DiscoveryApi;
+    identityApi: IdentityApi;
+  }) {
+    this.discoveryApi = options.discoveryApi;
+    this.identityApi = options.identityApi;
+  }
+
+  public async getConfig(): Promise<ConfigInfo | undefined> {
+    const urlSegment = 'config';
+
+    const configInfo = await this.get<ConfigInfo | undefined>(urlSegment);
+    return configInfo;
+  }
+
+  public async getExternalDependencies(): Promise<
+    ExternalDependency[] | undefined
+  > {
+    const urlSegment = 'external-dependencies';
```

Comment:
I guess this is more of a codestyle thing but i kind of prefer `this.get('/external-dependencies')` instead. But it's no biggie.

Reply:
Yeah, I'm cool with that. This has been cleaned up 👍